ark/utils/load_utils.py

```diff
@@ -131,7 +131,7 @@ def load_imgs_from_tree(data_dir, img_sub_folder=None, fovs=None, channels=None,
     if channels is None:
         channels = iou.list_files(
             dir_name=os.path.join(data_dir, fovs[0], img_sub_folder),
-            substrs=['.tif', '.jpg', '.png']
+            substrs=['.tif', '.tiff', '.jpg', '.png']
```

Comment:
Should we remove `tif` as an option? Is this going to lead to issues elsewhere if the user provides single-f files? Or it won't matter and it's fine to keep it?

Reply:
I think we can remove it, `.tif` should only be kept for MIBItiff backwards compatibility and they won't be using this function (or `load_imgs_from_dir`) to load in anything.